apps/src/templates/progress/progressHelpers.js

```diff
@@ -7,13 +7,13 @@ import { isHiddenForSection } from '@cdo/apps/code-studio/hiddenStageRedux';
  * particular lesson is visible, or hidden entirely.
  * @param {number} lesson - the lesson we're querying
  * @param {object} state - State of our entire redux store
- * @param {ViewType?} viewAs - Optional param to determine whether the lesson
- *   would be visible if viewing as someone else
+ * @param {ViewType} viewAs - Are we interested in whether the lesson is viewable
+ *   for students or teachers
  * @returns {boolean} True if the provided lesson is visible
  */
 export function lessonIsVisible(lesson, state, viewAs) {
```

Comment:
Previously we would call this method sometimes providing a viewAs, and sometimes not (in which case we would default to whatever was currently in the redux store). This was confusing, and I never really liked it. Now that SummaryProgressRow has viewAs as a prop (which will be the same as whatever is in the redux store), it's cleaner to just always pass in the viewAs prop and not have this as an optional param.